base/bin/build-env

```diff
@@ -18,7 +18,7 @@ DEBUG=${DEBUG:-0}  # `DEBUG=1 build-env` to run with debugging turned ON
 DOCKER_HOST_TUNNEL=localhost:2374
 GIT_USER_EMAIL=${GIT_USER_EMAIL:-ci@docksal.io}
 GIT_USER_NAME=${GIT_USER_NAME:-Docksal CI}
-REMOTE_BUILD_DIR_CLEANUP=${REMOTE_BUILD_DIR_CLEANUP:-1} # Default to re-initializing environment.
```

Comment:
`REMOTE_BUILD_DIR_CLEANUP` is not used in `build-env` at all. This line should just be moved to `build-init`.

Reply:
All set.